tests/unit/models/reddit/test_subreddit.py

```diff
@@ -1,7 +1,10 @@
+import json
+import os
 import pickle
-
```

Comment:
super super minor, but let's leave this line in to separate standard library packages from third-party (though `unittest` is standard library so it can be in the first block).

Reply:
It doesn't matter too much, because isort will move the import statements around whenever it is ran and add lines automatically.

Reply:
Why not run isort then before making the PR?